### ISSUE TITLE
[bugfix-1.0.x] M115 report incorrectly formatted

### ISSUE THREAD

**Issue by andrivet** [MarlinFirmware/Marlin#10486]
### Description

The code in branch `bugfix-1.0.x` is incorrect and does not properly format `M115` reports (a column is missing)

### Steps to Reproduce

* Send a `M115` command to a firmware compiled with sources from branch `bugfix-1.0.x`.
* A column is missing (see below for a detailed description of the problem).

**Expected behavior:**

The code in branch `1.0.x` is correct and produces:

```
Cap:SERIAL_XON_XOFF:0
Cap:EEPROM:1
Cap:VOLUMETRIC:1
Cap:AUTOREPORT_TEMP:1
Cap:PROGRESS:0
Cap:PRINT_JOB:1
Cap:AUTOLEVEL:1
Cap:Z_PROBE:1
Cap:LEVELING_DATA:1
Cap:BUILD_PERCENT:0
Cap:SOFTWARE_POWER:0
Cap:TOGGLE_LIGHTS:0
Cap:CASE_LIGHT_BRIGHTNESS:0
Cap:EMERGENCY_PARSER:1
```

**Actual behavior:**

In branch `bugfix-1.0.x`, `M115` outputs a report such as: 

```
Cap:SERIAL_XON_XOFF0
Cap:EEPROM1
Cap:VOLUMETRIC1
Cap:AUTOREPORT_TEMP1
Cap:PROGRESS0
Cap:PRINT_JOB1
Cap:AUTOLEVEL1
Cap:Z_PROBE1
Cap:LEVELING_DATA1
Cap:BUILD_PERCENT0
Cap:SOFTWARE_POWER0
Cap:TOGGLE_LIGHTS0
Cap:CASE_LIGHT_BRIGHTNESS0
Cap:EMERGENCY_PARSER1
```

#### Additional Information

The code in branch `bugfix-1.0.x` is the following:

```
#if ENABLED(EXTENDED_CAPABILITIES_REPORT)
  static void cap_line(const char * const name, bool ena=false) {
    SERIAL_PROTOCOLPGM("Cap:");
    serialprintPGM(name);
    SERIAL_PROTOCOLLN(int(ena ? 1 : 0));
  }
#endif
```

In branch `1.0.x`, it is:

```
#if ENABLED(EXTENDED_CAPABILITIES_REPORT)
  static void cap_line(const char * const name, bool ena=false) {
    SERIAL_PROTOCOLPGM("Cap:");
    serialprintPGM(name);
    SERIAL_PROTOCOLPGM(":");
    SERIAL_PROTOCOLLN(int(ena ? 1 : 0));
  }
#endif
```

In other terms, the line:

```
SERIAL_PROTOCOLPGM(":");
```

is missing in branch `bugfix-1.0.x`.

I can send you a PR is you like.

**Comment by andrivet** [comment]
To add some information:

The code in branch `1.0.x` was fixed on December 31, 2017 by the commit `21ba70d58739a24a988db8273ab997139653dec6`: Fixups for 1.1.8

**Comment by andrivet** [comment]
Sorry, I look at a old version of the sources. It has also been fixed in bugfix-1.0.x.
